embeddingapi/embedding-asyncapi-android-tests/embeddingapi/src/org/xwalk/embedding/base/XWalkViewTestBase.java

```diff
@@ -123,7 +123,8 @@ public XWalkViewTestBase() {
     protected void setUp() throws Exception {
         super.setUp();
         mainActivity = (MainActivity) getActivity();
-	mWebServer = TestWebServer.start();
+        mWebServer = TestWebServer.start();
+        mWebServerSsl = TestWebServer.startSsl();
```

Comment:
We need add website testing. such http://www.baidu.com(no  ssl certification) and https://github.com/(with  ssl certification)